### PR TITLE
Add loop control #250

### LINKED ISSUE #250: Trivial suggestion: add a checkbox to apply auto replay / loop
It's really a trivial feature I think some of us would like to have.

- Able to control wether to stop a song or just auto replay [infinite loop] the song by one simple checkbox when the song ended.
by the way, the app is really interesting.
Let me know what you think

### PR DESCRIPTION
**Pull request by andy-shi88**
Closes #250 

### Motivation and Context
- [ ] Why is this change required? What problem does it solve?
- [x] If it fixes an open issue, include the text `Closes #1` (where 1 would be the issue number) to your commit message.

### Types of changes
What types of changes does your code introduce? Check all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Description
- [x] Describe your changes in detail.
- We can loop through a song with this feature.

### Final checklist:
Go over all the following points and check all the boxes that apply  
If you're unsure about any of these, don't hesitate to ask. We're here to help!
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](.github/CONTRIBUTING.md) guidelines.
- [ ] All tests passed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -74,6 +74,9 @@ <h2 class="color-grey hide flash" id="mobile-message"></h2>
                 <button class="btn" id="toggleDescription">
                     Show Description
                 </button>
+                <input id="toggleLoop" name="toggleLoop" type="checkbox" />
+                <label id="toggleLoopLabel" for="toggleLoop">Loop</label>
```
Comment (shakeelmohamed) [inline index.html]: Maybe "Repeat track" is a better label?

File: css/styles.css [hunk 2]
```diff
@@ -57,6 +57,13 @@ a {
     margin: 10px auto 0;
 }
 
+#toggleLoopLabel{
```
Comment (shakeelmohamed) [inline css/styles.css]: Please conform to the CSS formatting we're already using, that's why Travis CI failed the build. You can click the details link for the specific errors

**Comment by andy-shi88** [comment]
I've pushed the new commits, let me know what you think.

File: js/everything.js [hunk 3]
```diff
@@ -651,6 +658,15 @@ $(function() {
         $("#demo").hide();
     }
 
+    $("input[name='toggleLoop']").change(function() {
+        if ($(this).is(":checked")) {
```
Comment (shakeelmohamed) [inline js/everything.js]: This if/else logic should be simplified to `ZenPlayer.isLoop = $(this).is(":checked");`

File: index.html [hunk 4]
```diff
@@ -74,9 +74,12 @@ <h2 class="color-grey hide flash" id="mobile-message"></h2>
                 <button class="btn" id="toggleDescription">
                     Show Description
                 </button>
+                <input id="toggleLoop" name="toggleLoop" type="checkbox" />
+                <label id="toggleLoopLabel" for="toggleLoop">Repeat track</label>
+
             </div>
 
-            <!-- Plyr controls' icons and Plyr player itself -->
+            <!-- Plyr controls' icons and Plyr player itself -->
```
Comment (shakeelmohamed) [inline index.html]: nit: trailing space

File: index.html [hunk 5]
```diff
@@ -74,9 +74,12 @@ <h2 class="color-grey hide flash" id="mobile-message"></h2>
                 <button class="btn" id="toggleDescription">
                     Show Description
                 </button>
+                <input id="toggleLoop" name="toggleLoop" type="checkbox" />
+                <label id="toggleLoopLabel" for="toggleLoop">Repeat track</label>
+
```
Comment (shakeelmohamed) [inline index.html]: nit: blank line

File: index.html [hunk 6]
```diff
@@ -74,9 +74,12 @@ <h2 class="color-grey hide flash" id="mobile-message"></h2>
                 <button class="btn" id="toggleDescription">
                     Show Description
                 </button>
+                <input id="toggleLoop" name="toggleLoop" type="checkbox" />
+                <label id="toggleLoopLabel" for="toggleLoop">Repeat track</label>
```
Comment (shakeelmohamed) [inline index.html]: Let's rename the id to `toggleRepeatLabel`

File: index.html [hunk 7]
```diff
@@ -74,9 +74,12 @@ <h2 class="color-grey hide flash" id="mobile-message"></h2>
                 <button class="btn" id="toggleDescription">
                     Show Description
                 </button>
+                <input id="toggleLoop" name="toggleLoop" type="checkbox" />
```
Comment (shakeelmohamed) [inline index.html]: Let's rename the id to `toggleRepeat`

File: css/styles.css [hunk 8]
```diff
@@ -57,6 +57,14 @@ a {
     margin: 10px auto 0;
 }
 
+#toggleLoopLabel {
```
Comment (shakeelmohamed) [inline css/styles.css]: rename, see below

File: js/everything.js [hunk 9]
```diff
@@ -651,6 +658,15 @@ $(function() {
         $("#demo").hide();
     }
 
+    $("input[name='toggleLoop']").change(function() {
```
Comment (shakeelmohamed) [inline js/everything.js]: rename, see above

File: index.html (hunk 4 above)
Comment (andy-shi88) [inline index.html]: weirly there is no trailling space on my side.

**Comment by andy-shi88** [comment]
done, feel free to recheck

**Comment by shakeelmohamed** [comment]
Thanks @andy-shi88

**Pull request merged**